smarty/environment.py

```diff
@@ -226,90 +262,50 @@ def asSMIRKS(self):
             """
             Returns
             --------
-            the same as asSMARTS()
-                for consistency asSMARTS() or asSMIRKS() can be called
-                for all environment objects
+            smarts : str
+                The SMIRKS string for just this bond
             """
+            #the same as asSMARTS()
+            #    for consistency asSMARTS() or asSMIRKS() can be called
+            #    for all environment objects
             return self.asSMARTS()
 
-        def addORtype(self, ORtype):
-            """
-            Adds ORtype to the set for this bond.
-
-            Parameters
-            --------
-            ORtype: string
-                added to the list of ORtypes for this bond
-            """
-            self.ORtypes.append(ORtype)
-
-        def addANDtype(self, ANDtype):
-            """
-            Adds ANDtype to the set for this bond.
-
-            Parameters
-            --------
-            ANDtype: string
-                added to the list of ANDtype for this bond
-            """
-            self.ANDtypes.append(ANDtype)
-
-        def getORtypes(self):
-            """
-            returns a copy of the list of ORtypes for this bond
+        def getOrder(self):
```

Comment:
Curious as I don't see this used anywhere -- what's the purpose of this function? (And why do "not triple" and "not aromatic" get bond orders of 1?)